test/spec/ol/index.test.js

```diff
@@ -0,0 +1,23 @@
+goog.provide('ol.test.index');
+goog.require('ol');
```

Comment:
Not entirely sure what to `goog.provide` for `index.js`, should it be just `ol.test`?